n2y/notion.py

```diff
@@ -391,12 +391,18 @@ def append_child_notion_blocks(self, block_id, children):
         def append_blocks(i1, i2, blocks, list):
             if i1 < i2:
                 child_list = blocks[i1:i2]
-                response = requests.patch(
-                    f"{self.base_url}blocks/{block_id}/children",
-                    json={"children": child_list}, headers=self.headers
-                )
-                appension_return = self._parse_response(response)
-                list.extend(appension_return['results'])
+                length = len(child_list)
+                for i in range(0, length, 100):
```

Comment:
For clarity, maybe pull 100 out int a variable, e.g., `max_new_blocks = 100` and use in in all three places where 100 is used?